app/src/AppRoutes.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Suspense } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Loading from 'components/common/Loading';
+import { Layout } from 'components/layout';
+
+const LazyAuthPage = React.lazy(() => import('components/auth/AuthPage'));
+const LazyLoopPage = React.lazy(() => import('components/loop/LoopPage'));
+const LazyHistoryPage = React.lazy(() => import('components/history/HistoryPage'));
+const LazyPoolPage = React.lazy(() => import('components/pool/PoolPage'));
+const LazySettingsPage = React.lazy(() => import('components/settings/SettingsPage'));
+const LazyConnectPage = React.lazy(() => import('components/connect/ConnectPage'));
+
+const AppRoutes: React.FC = () => {
+  return (
+    <Suspense fallback={<Loading delay={500} />}>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<LazyAuthPage />} />
```

Comment:
Think I would have to support `{PUBLIC_URL}`. Will look into whether I can specify a base app path and have other routes be relative to it. Iirc Angular used to support this as you would have SPAs nested deep in your site.

Reply:
I didn't fully test the `PUBLIC_URL` env var but will circle back to it in a follow-up review. This code looks like it should be fine though.